dist/source/repo-creator/choose-repository.ts

```diff
@@ -224,8 +233,12 @@ export class ChooseRepository {
 
 	private fetchPopular = (): void => {
 		this.repoCreator.getPopular().then((repos: RepositoryWireModel[]) => {
-			let popularTemplates = underscore(repos).map((repo: RepositoryWireModel) => new Repository(repo.owner, repo.name, false, false, true, false));
-			this.mergeTemplates(popularTemplates);
+			//let popularTemplates = underscore(repos).map((repo: RepositoryWireModel) => new Repository(repo.owner, repo.name, false, false, true, false));
+			//this.mergeTemplates(popularTemplates);
+			//let popularTemplates = underscore([{provider:'GitHub',owner:'Foo',name:'Bar'}]).map((repo: RepositoryWireModel) => new Repository(repo.owner, repo.name, false, false, true, false));
+		    //this.mergeTemplates(popularTemplates);
+			let popularTemplates = underscore([{provider:"GitHub",owner:"test",name:"Templates-Aurelia-TypeScript-Templates-Aurelia-TypeScript-Templates-Aurelia-TypeScript-Templates-Aurelia-TypeScript-Templates-Aurelia-TypeScript-Templates-Aurelia-TypeScript-Templates-Aurelia-TypeScript "}]).map((repo: RepositoryWireModel) => new Repository(repo.owner, repo.name, false, false, true, false));
+		    this.mergeTemplates(popularTemplates);
```

Comment:
This shouldn't be here actually, uncomment the previous and delete this in the PR.